### DISCUSSION TITLE
Generate single encounter with outpatient treatment

### DISCUSSION THREAD

**Discussion by dkozlovskyi** [synthetichealth/synthea#1498] [Q&A]
I'm trying to understand how I can generate a single encounter that ends up with an outpatient treatment plan. In other words how I can limit generated resources to just one encounter with the "outpatient" type and specific associated resources like MedicalRequest, CarePlan, and Procedure?

Appreciate your help

**Comment by jawalonoski** [comment]
Synthea is not designed to generate a single encounter.

If you only want a single outpatient encounter with 3 resources, you could write that by hand.

The purpose of the software is to create relatively complete longitudinal health records for an entire patient population.

Nonetheless, if you wanted to use Synthea for a single encounter, you would run the software, exporting the data format of your choice (default is HL7 FHIR), and then post-process the data. For example, filtering out all the encounters and data you don't want.

Alternatively, you could use the Module Builder to design a module that doesn't do anything but add an outpatient encounter with the resources you want, then remove all the other disease modules... but I would consider that a usage anti-pattern.

**Comment by dkozlovskyi** [comment]
Got it. Thanks for the explanations!
